bls/src/main/java/tech/pegasys/teku/bls/BLS.java

```diff
@@ -80,6 +91,13 @@ public static boolean verify(BLSPublicKey publicKey, Bytes message, BLSSignature
     return signature.getSignature().verify(publicKey.getPublicKey(), message);
   }
 
+  public static BLSPublicKey aggregatePublicKeys(List<BLSPublicKey> publicKeys) {
+    return new BLSPublicKey(
+        getBlsImpl()
+            .aggregatePublicKeys(
+                publicKeys.stream().map(BLSPublicKey::getPublicKey).collect(Collectors.toList())));
+  }
+
```

Comment:
As per the code comment higher up the file, my intention was that this BLS class follows [the standard](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-02) as closely as possible. There is no function similar to `aggregatePublicKeys` in the standard, and it seems only to be used in testing, so suggest moving it to a test class.

Reply:
Honestly speaking I would love to leave `aggregatePublicKeys` public assuming that our `bls` module can potentially be used by other projects. What do think of making this method static `PublicKey.aggregate()` and then I could rollback `aggregateSignatures` name

Reply:
Well... imo other projects should be encouraged to follow the BLS standard, in which case they don't need `aggregatePublicKeys` (except maybe for testing).  Your proposal is fine, though.

Reply:
Just though of one possible use case: if a hypothetical 'BLSCoin' has public keys == addresses there could be multisig addresses created by signers addresses aggregation :)

Reply:
Done: b60a0461